### PR TITLE
8.15.1 Release Notes

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/5493

Previews:
- [Index page](https://security-docs_bk_5748.docs-preview.app.elstc.co/guide/en/security/master/release-notes.html)
- [Release notes](https://security-docs_bk_5748.docs-preview.app.elstc.co/guide/en/security/master/release-notes-header-8.15.0.html#release-notes-8.15.1)

### CODE REVIEW

**Review by P1llus — APPROVED** [review]

File: docs/release-notes/8.15.asciidoc
Comment (gabriellandau) [inline docs/release-notes/8.15.asciidoc:28]:
```suggestion
* Windows Image load events now include process protection status, making it easier to detect both legitimate and malicious PPL activity.
```

File: docs/release-notes/8.15.asciidoc
Comment (gabriellandau) [inline docs/release-notes/8.15.asciidoc:42]:
```suggestion
* Improved CPU usage for Windows process events when the same executable is repeatedly launched, such as during compilation workloads.
```

**Review by gabriellandau — COMMENTED** [review]
I didn't realize that the Defend release notes would be intermixed with non-Defend notes.  

We should probably clarify which ones pertain to Defend.  Maybe prefix them with "[Defend]" or something?

File: docs/release-notes/8.15.asciidoc
Comment (nastasha-solomon) [inline docs/release-notes/8.15.asciidoc:49]: We might need to define this acronym. @gabriellandau can you verify that the definition is correct?

```suggestion
* Fixes an {elastic-defend} bug where Endpoint sometimes missed file and network events on newer kernels that support Extended Berkeley Packet Filter (eBPF). This only occurs if Endpoint fails to enable eBPF probes, and falls back to Kprobes. This fix is for Linux endpoints only.
```

**Review by tomsonpl — DISMISSED** [review]
Jamf analyzer 👍 LGTM, thanks!

**Review by logeekal — DISMISSED** [review]
Thanks @nastasha-solomon , changes look good from investigations side.

File: docs/release-notes/8.15.asciidoc
Comment (gabriellandau) [reply docs/release-notes/8.15.asciidoc:49]: I'm okay either way.  I think users technical enough to dig through our logs to identify the issue would know what ebpf is.  That being said, defining it here is harmless can save curious users the double click -> right click -> "Search Google"

**Review by gabriellandau — DISMISSED** [review]
LGTM.  Thanks @nastasha-solomon.

**Review by andrew-goldstein — APPROVED** [review]

File: docs/release-notes/8.15.asciidoc
Comment (nastasha-solomon) [reply docs/release-notes/8.15.asciidoc:49]: I'll hold off on defining the acronym for now and will only commit the tense-related edits that I added above. Thanks, @gabriellandau

**Review by natasha-moore-elastic — APPROVED** [review]
Entity Analytics notes LGTM!

**Review by gabriellandau — DISMISSED** [review]

**Review by andrew-goldstein — DISMISSED** [review]

**Review by yctercero — DISMISSED** [review]
LGTM! Thanks so much!

File: docs/release-notes/8.15.asciidoc
Comment (jmikell821) [inline docs/release-notes/8.15.asciidoc:51]:
```suggestion
* Fixes a bug that prevented duplicated prebuilt rules from inheriting **Required fields** and **Related integrations** field values ({kibana-pull}191065[#191065]).
```

File: docs/release-notes/8.15.asciidoc
Comment (jmikell821) [inline docs/release-notes/8.15.asciidoc:54]:
```suggestion
* Fixes a bug that hid the option to select a connector for Elastic AI Assistant ({kibana-pull}189944[#189944]).
```

**Review by jmikell821 — COMMENTED** [review]
A few fixes; would like to give a quick glance again before we merge. Thanks!

File: docs/release-notes/8.15.asciidoc
Comment (nastasha-solomon) [reply docs/release-notes/8.15.asciidoc:35]: Thanks, both! @gabriellandau I'm making one a minor edit to your suggested revision below to remove the dash after "recently" (that was my fault). Everything else looks good, thanks!

```suggestion
* Improves {elastic-defend} by reducing Malware Protection I/O disk and CPU usage when recently written files are subsequently executed. This update is for Windows endpoints only.
```

Comment (gabriellandau) [reply docs/release-notes/8.15.asciidoc]: We need to put `disk` before `I/O`.

File: docs/release-notes/8.15.asciidoc
Comment (gabriellandau) [inline docs/release-notes/8.15.asciidoc:35]:
```suggestion
* Improves {elastic-defend} by reducing Malware Protection disk I/O and CPU usage when recently written files are subsequently executed. This update is for Windows endpoints only.
```

File: docs/release-notes/8.15.asciidoc
Comment (nastasha-solomon) [reply docs/release-notes/8.15.asciidoc]: Oops, I accidentally pasted in the wrong order. Thank you - fixed!

File: docs/release-notes/8.15.asciidoc
Comment (jmikell821) [inline docs/release-notes/8.15.asciidoc:47]:
```suggestion
* Fixes an {elastic-defend} bug that can lead to {elastic-endpoint} reporting `STATUS_ACCESS_DENIED` when attempting to open files for `GENERIC_READ`. {elastic-endpoint} almost always recovered from this issue, but with this fix, it succeeds on the first try. This fix is for Windows endpoints only.
```

File: docs/release-notes/8.15.asciidoc
Comment (jmikell821) [inline docs/release-notes/8.15.asciidoc:50]:
```suggestion
* Fixes a bug that caused errors if you used Azure OpenAI connector for streaming ({kibana-pull}191552[#191552]).
```

**Review by jmikell821 — COMMENTED** [review]
Two small nits, then this will be good to merge.

**Review by joepeeples — APPROVED** [review]

**Pull request merged**
